go/vt/vtctl/workflow/vexec/vexec.go

```diff
@@ -42,6 +46,9 @@ const (
 	// VReplicationTableName is the unqualified name of the vreplication table
 	// supported by vexec.
 	VReplicationTableName = "vreplication"
+
+	// Timeout for callback function executions.
+	execTimeout = 10 * time.Second
```

Comment:
probably should make this configurable, right?

Reply:
Maybe. It wasn't even a variable before in wrangler. We can make it a parameter later. Or now if you want. I don't feel strongly.

Reply:
Changed it to use the ctx passed, which makes sense here (copy pasta oversight from wrangler).